packages/ui-extensions-react/src/surfaces/customer-account/hooks/attributes.ts

```diff
@@ -5,14 +5,21 @@ import type {
 
 import {useApi} from './api';
 import {useSubscription} from './subscription';
+import {ExtensionHasNoFieldError} from '../errors';
 
 /**
  * Returns the proposed `attributes` applied to the checkout.
  */
 export function useAttributes<
   Target extends RenderExtensionTarget = RenderExtensionTarget,
```

Comment:
hm... we limited it to only `RenderOrderStatusExtensionTarget` to strongly suggest that it should only be used on order status page, why do we change it back to RenderExtensionTarget as it will not put any type check error for targets outside order status page ?   https://github.com/Shopify/ui-extensions/blob/internal/packages/ui-extensions-react/src/surfaces/customer-account/hooks/attributes.ts#L24

Reply:
good catch!